src/endToEndHmpb.test.ts

```diff
@@ -238,8 +238,14 @@ test('failed scan with QR code can be adjudicated and exported', async () => {
     expect(JSON.parse(status.text).batches[0].count).toBe(expectedSampleBallots)
   }
 
+  const { id } = await store.dbGetAsync<{ id: string }>(`
+    select id
+    from ballots
+    where json_extract(metadata_json, '$.pageNumber') = 3
```

Comment:
you're confident the database is clean? Otherwise this might fail.

Reply:
Yeah, every test gets a new in-memory store: https://github.com/votingworks/module-scan/blob/3b3d051bb4696b0619ea442a5ecd12c4a0e9cd2b/src/endToEndHmpb.test.ts#L51